src/poker_now_log_converter/game.py

```diff
@@ -106,7 +107,69 @@ def __init__(self, poker_now_log: List[List[str]], currency: str = "USD", timezo
+                current_hand_street_max_bet = bet_amount
+                prev_action_dict[player_name_with_id] = bet_amount
+
+                current_hand.is_bomb_pot = True
+                current_hand.pre_flop_actions.append(Action(player=p_obj, bet_amount=bet_amount, action="bet"))
+
+                # Update button and blind positions based on the UTG player
+                utg_seat = current_hand.get_seat_by_player_name_with_id(player_name_with_id)
+                utg_seat_number = utg_seat.seat_number
+
+                # Clear previous blind and button information
+                current_hand.dealer = None
+                current_hand.small_blind_seat = None
+                current_hand.small_blind_player = None
+                current_hand.small_blind_amount = 0
+                current_hand.big_blind_seats = []
+                current_hand.big_blind_players = []
+                current_hand.big_blind_amount = 0
+
+                # Update button and blind positions
+                num_seats = len(current_hand.seats)
+                available_seats = [seat for seat in current_hand.seats if seat.stack_size > 0]
+
+                if available_seats:
+                    utg_seat = available_seats[0]
+                    utg_seat_number = utg_seat.seat_number
+
+                    button_seat_number = (utg_seat_number - 2) % num_seats
+                    small_blind_seat_number = (utg_seat_number - 1) % num_seats
+                    big_blind_seat_number = utg_seat_number
+
+                    button_seat = next(
+                        (seat for seat in current_hand.seats if seat.seat_number == button_seat_number), None)
+                    small_blind_seat = next(
+                        (seat for seat in current_hand.seats if seat.seat_number == small_blind_seat_number), None)
+                    big_blind_seat = next(
+                        (seat for seat in current_hand.seats if seat.seat_number == big_blind_seat_number), None)
+
+                    if button_seat:
+                        current_hand.dealer = button_seat.seat_player
+                    if small_blind_seat:
+                        current_hand.small_blind_seat = small_blind_seat
+                        current_hand.small_blind_player = small_blind_seat.seat_player
+                    if big_blind_seat:
+                        current_hand.big_blind_seats = [big_blind_seat]
+                        current_hand.big_blind_players = [big_blind_seat.seat_player]
+                else:
+                    # If no available seats, assume button is at seat 1
+                    button_seat = next((seat for seat in current_hand.seats if seat.seat_number == 1), None)
+                    if button_seat:
+                        current_hand.dealer = button_seat.seat_player
+                # current_hand.dealer = button_seat.seat_player
+                # current_hand.small_blind_seat = small_blind_seat
```

Comment:
Nit, if we go for another change before merging removing these would be nice, not a blocker though I can do this another time.